api/lib/domain/events/handle-complementary-certifications-scoring.js

```diff
@@ -55,19 +55,23 @@ async function handleComplementaryCertificationsScoring({
     minimumEarnedPix,
   });
 
-  const { computedComplementaryCertificationBadgeId, isAcquired } = await _getComplementaryCertificationResultInfo({
-    complementaryCertificationScoring,
-    hasComplementaryReferential,
-    complementaryCertificationBadgesRepository,
-    assessmentResult,
-    minimumReproducibilityRateLowerLevel,
-  });
+  const { computedComplementaryCertificationBadgeId, isAcquired, lowerLevelComplementaryCertificationBadgeAcquired } =
+    await _getComplementaryCertificationResultInfo({
```

Comment:
Ce n'est pas toi qui as créé cette méthode mais si on peut éviter les mots tronqués 🙏

```suggestion
    await _getComplementaryCertificationResultInformation({
```